### ISSUE TITLE
Parallel build fails

### ISSUE THREAD

**Issue by rosvall** [fwts/fwts#7]
Building fails with `make -j30`, but is fine with `make -j1`.

I apologize for the low-effort bug report. I'm a bit allergic to autotools etc.

```
~/src/fwts % make -j30
make  all-recursive
make[1]: Entering directory '/home/andreas/src/fwts'
Making all in src
make[2]: Entering directory '/home/andreas/src/fwts/src'
Making all in libfwtsiasl
make[3]: Entering directory '/home/andreas/src/fwts/src/libfwtsiasl'
m4 -P -I../../src/acpica/source/compiler ../../src/acpica/source/compiler/aslparser.y > aslcompiler.y
flex -i -PAslCompiler -oaslcompilerlex.c ../../src/acpica/source/compiler/aslcompiler.l
flex -i -PDtCompilerParser -odtcompilerparserlex.c ../../src/acpica/source/compiler/dtcompilerparser.l
flex -i -PDtCompilerParser -odtcompilerparserlex.c ../../src/acpica/source/compiler/dtcompilerparser.l
flex -i -PDtCompilerParser -odtcompilerparserlex.c ../../src/acpica/source/compiler/dtcompilerparser.l
flex -i -PDtParser -odtparserlex.c ../../src/acpica/source/compiler/dtparser.l
flex -i -PDtParser -odtparserlex.c ../../src/acpica/source/compiler/dtparser.l
flex -i -PDtParser -odtparserlex.c ../../src/acpica/source/compiler/dtparser.l
flex -i -PPrParser -oprparserlex.c ../../src/acpica/source/compiler/prparser.l
flex -i -PPrParser -oprparserlex.c ../../src/acpica/source/compiler/prparser.l
flex -i -PPrParser -oprparserlex.c ../../src/acpica/source/compiler/prparser.l
bison -y -d -bprparser -pPrParser ../../src/acpica/source/compiler/prparser.y
bison -y -d -bprparser -pPrParser ../../src/acpica/source/compiler/prparser.y
bison -y -d -bdtcompilerparser -pDtCompilerParser ../../src/acpica/source/compiler/dtcompilerparser.y
bison -y -d -bdtcompilerparser -pDtCompilerParser ../../src/acpica/source/compiler/dtcompilerparser.y
bison -y -d -bdtcompilerparser -pDtCompilerParser ../../src/acpica/source/compiler/dtcompilerparser.y
bison -y -d -bdtparser -pDtParser ../../src/acpica/source/compiler/dtparser.y
bison -y -d -bprparser -pPrParser ../../src/acpica/source/compiler/prparser.y
bison -y -d -bdtparser -pDtParser ../../src/acpica/source/compiler/dtparser.y
bison -y -d -bdtparser -pDtParser ../../src/acpica/source/compiler/dtparser.y
bison -y -d -d -baslcompiler -pAslCompiler aslcompiler.y
bison -y -d -d -baslcompiler -pAslCompiler aslcompiler.y
aslcompiler.y:211.1-7: warning: POSIX Yacc does not support %expect [-Wyacc]
  211 | %expect 128
      | ^~~~~~~
aslcompiler.y:211.1-7: warning: POSIX Yacc does not support %expect [-Wyacc]
  211 | %expect 128
      | ^~~~~~~
mv dtcompilerparser.tab.c dtcompilerparser.c
mv dtcompilerparser.tab.c dtcompilerparser.c
cp dtcompilerparser.tab.h dtcompilerparser.y.h
mv: cannot stat 'dtcompilerparser.tab.c': No such file or directory
make[3]: *** [Makefile:4216: dtcompilerparserlex.c] Error 1
make[3]: *** Waiting for unfinished jobs....
mv dtcompilerparser.tab.c dtcompilerparser.c
mv: cannot stat 'dtcompilerparser.tab.c': No such file or directory
make[3]: *** [Makefile:4215: dtcompilerparser.c] Error 1
mv dtparser.tab.c dtparser.c
mv prparser.tab.c prparser.c
cp dtparser.tab.h dtparser.y.h
mv prparser.tab.c prparser.c
cp prparser.tab.h prparser.y.h
mv: cannot stat 'prparser.tab.c': No such file or directory
make[3]: *** [Makefile:4230: prparser.c] Error 1
mv dtparser.tab.c dtparser.c
mv: cannot stat 'dtparser.tab.c': No such file or directory
make[3]: *** [Makefile:4222: dtparser.c] Error 1
mv prparser.tab.c prparser.c
mv: cannot stat 'prparser.tab.c': No such file or directory
make[3]: *** [Makefile:4229: prparserlex.c] Error 1
mv dtparser.tab.c dtparser.c
mv: cannot stat 'dtparser.tab.c': No such file or directory
make[3]: *** [Makefile:4222: dtparser.y.h] Error 1
mv aslcompiler.tab.c aslcompiler.c
cp aslcompiler.tab.h aslcompiler.y.h
mv aslcompiler.tab.c aslcompiler.c
mv: cannot stat 'aslcompiler.tab.c': No such file or directory
make[3]: *** [Makefile:4208: aslcompiler.c] Error 1
make[3]: Leaving directory '/home/andreas/src/fwts/src/libfwtsiasl'
make[2]: *** [Makefile:5115: all-recursive] Error 1
make[2]: Leaving directory '/home/andreas/src/fwts/src'
make[1]: *** [Makefile:828: all-recursive] Error 1
make[1]: Leaving directory '/home/andreas/src/fwts'
make: *** [Makefile:759: all] Error 2
```

This was on:

- Arch Linux x86_64
- linux 6.6.1-zen1-1-zen  
- autoreconf (GNU Autoconf) 2.71
- GNU Make 4.4.1

**Comment by bcran** [comment]
I see the same problem with commit 7c15229cfb544dc423e4b8c44758b7b7e5271743 building on openSUSE Tumbleweed AArch64.

I have an Ampere Altra CPU with 64 cores. Trying to build with `-j40` fails while `-j8` works.

[typescript-parallel-make_j40.txt](https://github.com/fwts/fwts/files/15299873/typescript-parallel-make_j40.txt)

**Comment by jolivain** [comment]
Hi,
I proposed a patch to solve this issue. See:
https://lists.ubuntu.com/archives/fwts-devel/2024-May/013852.html
The commit log contains the explanations.

**Comment by Ivanhu5866** [comment]
We still use launchpad to trace bug, file a bug on Launchpad for this issue, 
https://bugs.launchpad.net/fwts/+bug/2069675

**Comment by Ivanhu5866** [comment]
fix applied in V24.07.00
